src/App.css

```diff
@@ -2,11 +2,68 @@ body {
   margin: 0;
   padding: 0;
   box-sizing: border-box;
+  background: rgba(245, 246, 250, 1);
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: rgb(255, 255, 255);
+  border-bottom: 3px solid rgb(173, 170, 170);
+  margin: 20px 30px;
+  padding: 10px 20px;
+  text-decoration: none;
```

Comment:
- [ ] Please let's make sure the basic CSS bad practices are not used ([article for support](https://speckyboy.com/good-bad-css-practices)) No repeating of the same rule. I think you have done this pretty literally in this file. Let's kindly double check file and fix related issues :wrench: it :+1: